src/main/kotlin/org/dashevo/dpp/util/Entropy.kt

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2020-present, Dash Core Team
+ *
+ * This source code is licensed under the MIT license found in the
+ * COPYING file in the root directory of this source tree.
+ */
+package org.dashevo.dpp.util
+
+import org.bitcoinj.core.Address
+import org.bitcoinj.core.AddressFormatException
+import org.bitcoinj.core.ECKey
+import org.bitcoinj.params.TestNet3Params
+
+class Entropy {
+    companion object {
+
+        fun generate() : String {
+            val privateKey = ECKey()
+            return Address.fromKey(TestNet3Params.get(), privateKey).toString()
```

Comment:
(Can probably be postponed)
Should we pass the NetworkParams to the generate method? 
If so, maybe we should add NetworkParams as member of the classes that call this method, or something else.

Reply:
In this case, the network doesn't really matter for generating a random string. 

js-dpp is using testnet.